stellarphot/utils/data/PS1_to_JC.json

```
@@ -0,0 +1,92 @@
+{
+    "name": "PAN-STARRS1 to Johnson/Cousins",
+    "reference": "https://doi.org/10.1088/0004-637x/750/2/99",
+    "from_system": {
+        "name": "PANSTARRS1",
+        "passbands": [
+            "gp1",
+            "rp1",
+            "ip1",
+            "zp1",
+            "yp1",
+            "wp1"
+            ]
+    },
+    "to_system": {
+        "name": "JC",
+        "passbands": [
+            "B",
+            "V",
+            "Rc",
+            "Ic"
+        ]
+    },
+    "transform_coefficients": {
+        "gp1,B":{
+            "name": "gp1 to B",
+            "from_passband": "gp1",
+            "to_passband": "B",
+            "polynomial_coefficients": [
+                0.212,
+                0.556,
+                0.034
+            ],
+            "residual": 0.032
```

According to Table 6 in Tonry et al (2012)
```suggestion
            "residual": 0.034
```